### ISSUE TITLE
Map marker not shown in production environment

### ISSUE THREAD

**Issue by Pijuli** [red-made/strapi-geodata#2]
Hey!
Awesome addon, thank you!

There's an issue with the map (leaflet) in production environment
**Production**
![image](https://github.com/user-attachments/assets/5fd19d80-4f78-4c15-ace1-a42a539c8402)

**Development**
![image](https://github.com/user-attachments/assets/9f49bc81-ec5c-47ce-9af7-a6a8c92a49ed)

I think it's self explanatory, but the url turns into absolute path. Obviously, the image isn't there.
Do you have any easy fix? 

Thank you!

**Comment by red-made** [comment]
Hi, this issue has been resolved in version 0.1.02. I switched to using a CDN for the marker icon to bypass restrictions. After updating, please ensure that you add the security middleware directive to allow connections from the image CDN. I’ve also updated the README to reflect this change.

```
"img-src": [
  "'self'",
  "blob:",
  "data:",
  "https://*.basemaps.cartocdn.com",
  "market-assets.strapi.io",
  "https://*.tile.openstreetmap.org",
  "https://unpkg.com/leaflet@1.9.4/dist/images/"
]
```

**Comment by Pijuli** [comment]
Looks good to me!
Thank you!
